mesos-commons/src/main/java/org/apache/mesos/collections/Maps.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.mesos.collections;
+
+import com.google.common.base.Predicate;
+
+import javax.annotation.Nonnull;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ */
+public class Maps {
+
```

Comment:
filterMapKeysThatStartWith should be private, right? For that matter, couldn't we combine both methods into one?